spring-integration-core/src/main/java/org/springframework/integration/router/AbstractMessageRouter.java

```diff
@@ -204,4 +202,20 @@ else if (this.logger.isDebugEnabled()) {
 		}
 	}
 
+	private boolean doSend(MessageChannel channel, final Message<?> messageToSend) {
+		try {
+			this.messagingTemplate.send(channel, messageToSend);
+			return true;
+		}
+		catch (MessagingException e) {
+			if (!this.ignoreSendFailures) {
+				throw e;
+			}
+			else if (this.logger.isDebugEnabled()) {
+				this.logger.debug(e);
```

Comment:
I think on reason to wrap into `isDebugEnabled()` since we don't do any logic with argument to log.